### PR TITLE
Add initial draft of preface

### PR DESCRIPTION
**Pull request by canyon289**
There was a great discussion about intended audience level in slack. This notebook is meant to capture that. Please let me know what you think

### CODE REVIEW

File: AnIntroduction/Preface.ipynb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Intended Audience\n",
+    "While ArviZ strives to provide an easy way for Bayesian practioners to create exploratory plots and statistics, the package and docs alone do not explain how to **use** the exploratory plots and statistics. This repository is aimed to fill that gap, and serve as more in depth explanaton to all the functionality in ArviZ. Ideally it will be a handy reference for Bayesian practitioners that are using probabilistic programming but may be unfamiliar with the various methods of model criticism.\n",
+    "\n",
+    "Specifically we strive to write this guide to be understandable to the same audience as **Statistical Rethinking**, written by Richard McElreath. This is not to say that more advanced Bayesian practitioners won't find benefit in the explanation though!"
```
Comment (aloctavodia) [inline AnIntroduction/Preface.ipynb]: I will remove this last paragraph. Previous paragraph already explain very well our target audience.

Comment (canyon289) [reply AnIntroduction/Preface.ipynb]: Agree that it is redundant and happy to change it.

I think we should have a paragraph though that if the user is interesting in how to construct models in pymc3, stan, or Computational Bayes in general we should add links to resources. I'll  take a run at it to see what you think

File: AnIntroduction/Preface.ipynb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+   "source": [
+    "## Intended Audience\n",
+    "While ArviZ strives to provide an easy way for Bayesian practioners to create exploratory plots and statistics, the package and docs alone do not explain how to **use** the exploratory plots and statistics. This repository is aimed to fill that gap, and serve as more in depth explanaton to all the functionality in ArviZ. Ideally it will be a handy reference for Bayesian practitioners that are using probabilistic programming but may be unfamiliar with the various methods of model criticism.\n",
+    "\n",
+    "For those interested in how to construct models either  PyMC3 or Pystan, both libraries documentation numerous example.\n",
+    "* [PyMC3](https://docs.pymc.io/)\n",
+    "* [PyStan](https://pystan.readthedocs.io/en/latest/)\n",
+    "* [CmdStan](http://mc-stan.org/users/interfaces/cmdstan)\n",
+    "\n",
+    "If interested in Baayesian Statistical methods in general the following resources are reccomended\n",
```
Comment (aloctavodia) [inline AnIntroduction/Preface.ipynb]: Typo Baayesian

File: AnIntroduction/Preface.ipynb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    "* [Bayesian Methods for Hackers](https://github.com/CamDavidsonPilon/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers)\n",
+    "* [Statistical Rethinking](https://www.amazon.com/Statistical-Rethinking-Bayesian-Examples-Chapman/dp/1482253445)\n"
```
[the remark this answers is not in the document]
Comment (canyon289) [reply AnIntroduction/Preface.ipynb]: I agree, I'll add in next PR and if anyone disagrees we can remove it :)

**Pull request merged**